.github/workflows/firebase_analytics.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: "Install Flutter"
-        run: ./.github/workflows/scripts/install-flutter.sh dev
```

Comment:
Why this change?

Reply:
@rrousselGit I had to switch to the stable channel because the iOS examples changed some properties that should not be changed, leading to building errors on the dev channel.

On the stable channel, it still works:
https://github.com/flutter/flutter/issues/78345

Should we rather fix the build issues and stick with the dev channel?
OR should we do it in this PR or a separate one?

Reply:
The CI has been fixed, so I'll revert this